xml/dhcp_vendor_class.xml

```diff
@@ -305,4 +354,38 @@
     <param pos="0" name="os.vendor" value="Cisco"/>
   </fingerprint>
 
-</fingerprints>
\ No newline at end of file
+  <fingerprint pattern="^BrightSign ((?:HS|XD|LS|XT|HD)\d{3,4}(?:-W|-PP)?)$">
+    <description>BrightSign Digital Signage Player</description>
+    <example hw.model="HS124">BrightSign HS124</example>
+    <example hw.model="XD1034">BrightSign XD1034</example>
+    <example hw.model="XD1034-W">BrightSign XD1034-W</example>
+    <example hw.model="LS424-W">BrightSign LS424-W</example>
+    <example hw.model="XT244">BrightSign XT244</example>
+    <example hw.model="HD1023">BrightSign HD1023</example>
+    <example hw.model="XT1144-PP">BrightSign XT1144-PP</example>
+    <param pos="0" name="hw.vendor" value="BrightSign"/>
+    <param pos="0" name="hw.device" value="Media Server"/>
+    <param pos="1" name="hw.model"/>
+    <param pos="0" name="os.vendor" value="BrightSign"/>
+    <param pos="0" name="os.family" value="Linux"/>
```

Comment:
Based on the BrightSign CPE that exists (`o:brightsign:4k242_firmware`) this should probably have an `os.product` param.
```
<param pos="0" name="os.product" value="{hw.model} Firmware"/>
```

Reply:
Will do.